### PR TITLE
Update ble_l2cap.c

### PR DESCRIPTION
**Pull request by Francois-Belanger**
Fix potential null pointer crash

### CODE REVIEW

File: nimble/host/src/ble_l2cap.c [hunk 1]
```diff
@@ -403,7 +403,9 @@ ble_l2cap_tx(struct ble_hs_conn *conn, struct ble_l2cap_chan *chan,
 
     case BLE_HS_EAGAIN:
         /* Controller could not accommodate full packet.  Enqueue remainder. */
-        STAILQ_INSERT_TAIL(&conn->bhc_tx_q, OS_MBUF_PKTHDR(txom), omp_next);
+        if (txom) {
+            STAILQ_INSERT_TAIL(&conn->bhc_tx_q, OS_MBUF_PKTHDR(txom), omp_next);
+        }
```
Comment (andrzej-kaczmarek) [inline nimble/host/src/ble_l2cap.c]: I did not look into code too much, but this does look like a workaround for an issue in in `ble_hs_hci_acl_tx` which should not return `BLE_HS_EAGAIN` and nullify `txom`. Basically with this fix we would return success to caller and drop packet.

Comment (rymanluk) [reply nimble/host/src/ble_l2cap.c]: @francois-berder any chances to get traces from that issue?

Comment (Francois-Belanger) [reply nimble/host/src/ble_l2cap.c]: Hi, will try to test within days.  Was on another project.

**Comment by Francois-Belanger** [comment]
Just used the latest idf, bug seems fixed.  Unable to reproduce :-)

IDF v4.1-dev-2224-gbcd2358eb

**Pull request closed without merging**